src/app/[lang]/documents/page.tsx

```diff
@@ -0,0 +1,14 @@
+"use client";
+import { useDictionary } from "@/contexts/dictionary-provider";
+
+export default function Documents() {
+  const dict = useDictionary();
+
+  return (
+    <main>
+      <h1 className="font-mediumpx-2 font-title text-3xl md:px-5">
```

Comment:
Please add the page padding on `<main>` and make sure you add a space between `font-medium px-2

```suggestion
    <main className="layout-p-full">
      <h1 className="font-medium px-2 font-title text-3xl md:px-5">
```